README.md

```diff
@@ -790,17 +564,16 @@ for time_step in range(steps): # simulation loop
     client.put_dataset(dataset)
 ```
 
-Finally, in `driver.py`, we can see that with minimal effort we are able to launch the
-database and run the simulation in a non-blocking fashion. From there we simply poll
-the Orchestrator for new data posted by our simulation and update our plot once it is
-received.
+The driver script, `driver.py`, launches the Orchestrator database and runs
+the simulation in a non-blocking fashion. The driver script then uses the SmartRedis
+client to pull the DataSet and plot the vorticity while the simulation is running.
 
 ```Python
 # driver.py
 time_steps, seed = 3000, 42
 
 exp = Experiment("finite_volume_simulation", launcher="local")
-db = Orchestrator(port=6780)
+db = exp.create_database(port=6780, interface="lo")
```

Comment:
This is the first mention/use of ``interface``.  Would it be helpful to have a comment on what that is?